doc/PackCreation.md

```diff
@@ -230,3 +230,47 @@ And what it looks like back in tracker.json to include that tab group of groups.
 ### **Finishing up**
 And now’s the point where you get to put your pieces into action, and see what’s broken. Copy your whole repository folder into poptracker’s Packs folder, open up Poptracker, and see how it looks. You can make changes to the files in the packs folder with Poptracker open and press the Refresh button to load your changes. From here, double-check that logic functions the way you expect it to when items are added / removed, confirm that your maps are readable, and make sure the layouts don’t waste a bunch of space – not everyone has a nice monitor to see tiny pixels.  
 If you created a variant (way back in the Manifest.json), then you’re going to need to create the files that get loaded in place of the default JSON files – but that’s really just more of what you’ve already been doing. Read the documentation in the example pack, and you’ll be good to go.
+
+
+
+# Poptracker Builder-Script

```

Comment:
```suggestion
## Poptracker Builder-Script
```

I approve >1 blank line, but more than 2 is never necessary. Also `#` is document title and `##` is highest level heading, so should probably be `##`